src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -353,16 +353,14 @@ private function reRegister($prev)
      * @param string $message
      * @param string $file
      * @param int    $line
-     * @param array  $context
-     * @param array  $backtrace
      *
      * @return bool Returns false when no handling happens so that the PHP engine can handle the error itself
      *
      * @throws \ErrorException When $this->thrownErrors requests so
      *
      * @internal
```

Comment:
Fortunately, this annotation is here from the beginning so we can change the signature without breaking our BC policy.